### ISSUE TITLE
val_loss can not decrease

### ISSUE THREAD

**Issue by Chung-Yi** [qjadud1994/CRNN-Keras#58]
I faced some problem when I trained. At the beginning, both the loss and val_loss decreased step by step, but then only loss decreased val_loss frozen between 24 and 25. My trained data is 1922 and my tested data for evaluate is 213, their ratio is 9(9:1), I have no idea if it is enough for training, and I have no idea if it is the reason cause the problem.

![擷取](https://user-images.githubusercontent.com/26489985/132269713-0d64ecdb-bc68-4a9f-9b20-0df03ead4527.JPG)

**Comment by Chung-Yi** [comment]
It seems that it was overfitting.

**Comment by Chung-Yi** [comment]
Does anyone have the same problem?
